bcda/auth/plugin/alpha.go

```diff
@@ -0,0 +1,43 @@
+package auth
+
+import (
+	"errors"
+)
+
+type AlphaAuthPlugin struct{}
+
+func (p *AlphaAuthPlugin) RegisterClient(params []byte) error {
+	return errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) UpdateClient(params []byte) error {
+	return errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) DeleteClient(params []byte) error {
+	return errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) GenerateClientCredentials(params []byte) ([]byte, error) {
+	return nil, errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) RevokeClientCredentials(params []byte) error {
+	return errors.New("Not yet implemented")
+}
+
+func (p *AlphaAuthPlugin) RequestAccessToken(params []byte) ([]byte, error) {
```

Comment:
As discussed below, I think the return value is a specific type, Token. The params are correct as []byte, because different providers of tokens might require different parameters to construct tokens. But a Token is a Token.